guides/common/modules/ref_predefined-roles.adoc

```diff
@@ -7,48 +7,140 @@ To view the exact set of permissions a predefined role grants, display the role
-| Compliance manager | View, create, edit, and destroy SCAP content files, compliance policies, and tailoring files.
+|
+
+|Ansible Tower Inventory Reader
+|View facts, hosts, and host groups.
+|
+
+|Bookmarks manager
+|Create, edit, and delete bookmarks.
+|
+
+|Boot disk access
+|Download the boot disk.
+|
+
+|Compliance manager
+|View, create, edit, and destroy SCAP content files, compliance policies, and tailoring files.
+
 View compliance reports.
-| Compliance viewer | View compliance reports.
-| Create ARF report | Create compliance reports.
-| Default role | The set of permissions that every user is granted, irrespective of any other roles.
-| Discovery Manager| View, provision, edit, and destroy discovered hosts and manage discovery rules.
-| Discovery Reader | View hosts and discovery rules.
-| Edit hosts | View, create, edit, destroy, and build hosts.
-| Edit partition tables | View, create, edit and destroy partition tables.
-| Manager | View and edit global settings.
-| Organization admin | An administrator role defined per organization.
+|
+
+|Compliance viewer
+|View compliance reports.
+|
+
+|Create ARF report
+|Create compliance reports.
+|
+
+|Default role
+|View tasks and jobs invocations.
+|{Project} automatically assigns this role to every user in the system.
+
+|Discovery Manager
+|View, provision, edit, and destroy discovered hosts.
+
+Manage discovery rules.
+|
+
+|Discovery Reader
+|View hosts and discovery rules.
+|
+
+|Edit hosts
+|View, create, edit, destroy, and build hosts.
+|
+
+|Edit partition tables
+|View, create, edit, and destroy partition tables.
+|
+
+|Manager
+|View and edit global settings.
+|
+
+|Organization admin
+|All permissions except permissions for managing organizations.
+|An administrator role defined per organization.
 The role has no visibility into resources in other organizations.
-| Red{nbsp}Hat Access Logs | View the log viewer and the logs.
-| Remote Execution Manager | Control which roles have permission to run infrastructure jobs.
-| Remote Execution User | Run remote execution jobs against hosts.
-| Site manager | A restrained version of the Manager role.
-| System admin a|
-* Edit global settings in *Administer* > *Settings*.
-* View, create, edit and destroy users, user groups, and roles.
-* View, create, edit, destroy, and assign organizations and locations but not view resources within them.
-
-Users with this role can create users and assign all roles to them.
+
+By cloning this role and assigning an organization, you can delegate administration of that organization to a user.
+
+|Red{nbsp}Hat Access Logs
+|View the log viewer and the logs.
+|
+
+|Remote Execution Manager
+|Control which roles have permission to run infrastructure jobs.
+|
+
+|Remote Execution User
+|Run remote execution jobs against hosts.
+|
+
+|Site manager
+|View hosts, users, logs, and other items.
+
+Manage hosts, users, installation media, subnets, and other items.
```

Comment:
Users with this role should be able to also view all the things they can manage. It is a little bit odd that subnets are mentioned in the list of things that can be managed, but not in the list of things that can be viewed.